paimon-web-server/src/test/java/org/apache/paimon/web/server/controller/CatalogControllerTest.java

```diff
@@ -0,0 +1,102 @@
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.server.controller;
+
+import org.apache.paimon.web.server.data.model.CatalogInfo;
+import org.apache.paimon.web.server.data.result.R;
+import org.apache.paimon.web.server.util.ObjectMapperUtils;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+import org.junit.jupiter.api.io.TempDir;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultHandlers;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+/** Test for CatalogController. */
+@SpringBootTest
+@AutoConfigureMockMvc
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public class CatalogControllerTest extends ControllerTestBase {
+
+    private static final String catalogPath = "/api/catalog";
+
+    @TempDir java.nio.file.Path tempFile;
+
+    private static final String catalogName = "testCatalog";
+
+    @Test
+    @Order(1)
+    public void testCreateCatalog() throws Exception {
+        CatalogInfo catalogInfo = new CatalogInfo();
+        catalogInfo.setCatalogType("filesystem");
+        catalogInfo.setCatalogName(catalogName);
+        catalogInfo.setWarehouse(tempFile.toUri().toString());
+        catalogInfo.setDelete(false);
+
+        String responseString =
+                mockMvc.perform(
+                                MockMvcRequestBuilders.post(catalogPath + "/createCatalog")
+                                        .cookie(cookie)
+                                        .content(ObjectMapperUtils.toJSON(catalogInfo))
+                                        .contentType(MediaType.APPLICATION_JSON_VALUE)
+                                        .accept(MediaType.APPLICATION_JSON_VALUE))
+                        .andExpect(MockMvcResultMatchers.status().isOk())
+                        .andDo(MockMvcResultHandlers.print())
+                        .andReturn()
+                        .getResponse()
+                        .getContentAsString();
+
+        R<Void> r = ObjectMapperUtils.fromJSON(responseString, new TypeReference<R<Void>>() {});
+        assertEquals(200, r.getCode());
+
+        mockMvc.perform(
```

Comment:
Any assert?

Reply:
assertEquals(200, r.getCode())： The current test verifies whether the interface is accessible and whether the parameters are correct.